src/locales/id.locale.json

```diff
@@ -202,6 +202,7 @@
   "mno_enterprise.templates.dashboard.menu.marketplace": "Pasar",
   "mno_enterprise.templates.dashboard.menu.logout": "Keluar",
   "mno_enterprise.templates.dashboard.menu.new_company": "Perusahaan Baru",
+  "mno_enterprise.templates.dashboard.menu.messages": "Pesan",
```

Comment:
You don't need to add a translation in those files. The default fallback language is `en.json`